vnext/PropertySheets/WinAppSDK.WinUI.props

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
do we need this file at all? I thought we could just use WinUI.props

Reply:
WinUI.props imports a different version of Microsoft.WindowsAppSDK.props which includes winmd files, this was as a workaround for that although I'm not sure if this entirely fixes that issue.

Reply:
I'm still not convinced we need this :) After this change we will only have a single way to create winui3 apps anyway, so i'm not sure why we would need windowsappsdk references in both winui.props and this new file. My proposal would be to ideally stick with the existing file; if not, then we should remove the winui3 logic from winui.props. But then I think we'd need more changes in app project templates

Reply:
Synced offline, added a check to existing WinUI.props instead to import the appropriate WindowsAppSDK.props depending on TargetFramework. Removing this file.